web/forms.py

```
@@ -746,6 +747,32 @@ def clean_username(self):
             raise forms.ValidationError("This username is already taken. Please choose a different one.")
         return username
 
+    def _clean_social_media_username(self, field_name, validator):
```

Ruff (0.15.6)

[warning] Missing return type annotation for private function `_clean_social_media_username`

(ANN202)